components/places/src/storage/bookmarks/root_guid.rs

```diff
@@ -61,15 +61,15 @@ impl BookmarkRootGuid {
         self.guid().clone()
     }
 
-    pub fn from_str(guid: &str) -> Option<Self> {
+    pub fn well_known(guid: &str) -> Option<Self> {
```

Comment:
Hmm, why didn't Clippy like `from_str`, but allowed `from_guid`? I liked the symmetry between `from_str` and `from_guid` before (and matching `as_str` and `as_guid`, too)...

Reply:
It insisted either we implement `std::str::FromStr` or rename it if we're going to call things from_str.

AFAICT FromStr requires we return a Result<T, impl std::error::Error>, so I just renamed it.